src/Message/AbstractRequest.php

```diff
@@ -73,6 +83,10 @@ protected function xml2array($xml)
     {
         $arr = [];
```

Comment:
Can we change this to a more meaningful name.
```suggestion
        $response = []; // or $result
```

Reply:
Sounds good to me!